ui/js/utils.js

```diff
@@ -0,0 +1,100 @@
+/**
```

Comment:
In ES6 it defaults to module namespace. That might be an issue if we don't like the calling style (`getLocal()` vs `utils.getLocal()`) but it's not going to pollute scope anywhere we don't want it to.